dev-support/Jenkinsfile_GitHub

```diff
@@ -156,11 +161,16 @@ pipeline {
                         }
                     }
                     environment {
+                        ////
                         // customized per parallel stage
-                        PLUGINS = "${JDK_SPECIFIC_PLUGINS}"
+                        //
+                        // run SpotBugs only once, and run it on JDK8.
+                        PLUGINS = "${JDK_SPECIFIC_PLUGINS},spotbugs"
```

Comment:
It used to be -findbugs with hyphen. No need of it when doing spotbugs?

Reply:
`-fingbugs` explicitly disabled findbugs. I'm trying to explicitly enable `spotbugs`. The two appear to be driven by the same yetus "check", so I'm attempting to ensure that module is enabled. Because we don't ship findbugs binaries in our Dockerfile anymore, I think that "check" will do the right thing, as long as it's enabled (see earlier comments re: that check also requiring the `maven` check as a dependency).

Reply:
The -fingbugs is from the release note of yetus, as the official way to enable spotbugs.

https://issues.apache.org/jira/browse/YETUS-749

Of course I haven't checked the code of yetus so maybe specify spotbugs also works.